packages/salesforcedx-vscode-apex/snippets/apex.json

```
@@ -410,5 +409,54 @@
             "}"
         ],
         "description": "Create a mock for an http callout for apex testing"
+    },
+    "Is Blank": {
+        "prefix": "isb",
+        "body": [
+            "String.isBlank(${0})"
+        ],
+        "description": "String isBlank"
+    },
+    "Is Not Blank": {
+        "prefix": "isnb",
+        "body": [
+            "String.isNotBlank(${0})"
```

Same reasoning as above - String.isNotBlank(inputString) is a method that takes one argument.

```suggestion
            "String.isNotBlank(${1:inputString})"
```